corehq/ex-submodules/phonelog/reports.py

```diff
@@ -262,8 +258,22 @@ def _create_rows(self, logs, matching_id=None, range=None):
                 '<i class="icon icon-info-sign"></i></a>'
             ) % (version.split(' ')[0], html.escape(version))
 
-            row_set.append([ui_date, log_tag_format, username_fmt,
-                            device_users_fmt, device_fmt, log.msg, ver_format])
+            return [ui_date, log_tag_format, username_fmt,
+                    device_users_fmt, device_fmt, log.msg, ver_format]
+
+    def _create_rows(self, logs, matching_id=None, range=None):
+        _device_users_by_xform = memoized(device_users_by_xform)
+        row_set = []
+        user_query = self._filter_query_by_slug(DeviceLogUsersFilter.slug)
+        device_query = self._filter_query_by_slug(DeviceLogDevicesFilter.slug)
+
+        self.total_records = logs.count()
+        logs = logs.order_by(self.ordering)
+        if range:
```

Comment:
Yeah, agreed. This was actually just a refactor so that I could add the domain element in this report. I don't think this code has been touched since like 2012 or something.